docs/codeql/index.html

```diff
@@ -39,7 +39,7 @@
                         CodeQL tools
                     </div>
                     <li><a class="dropdown-item" href="codeql-for-visual-studio-code">CodeQL for VS Code</a>
-                    <li><a class="dropdown-item" href="codeql-cli">CodeQL CLI</a>
+                    <li><a class="dropdown-item" href="https://docs.github.com/en/code-security/codeql-cli/getting-started-with-the-codeql-cli/about-the-codeql-cli">CodeQL CLI</a>
```

Comment:
How will this work for preview builds?

Reply:
In my local testing, it just works as expected and opens the page in Github user docs

Reply:
Wouldn't we prefer a relative link for a preview, so you go to the same page within the preview build?
(Not blocking for this PR, but we've run into this in the past.)

Reply:
This is a link from the CodeQL docs site to the GitHub user docs site. I think you may have missed this 🤔, otherwise I'm obviously missing something.

Reply:
I did indeed miss that! Thanks.